ui/app/src/components/UploadDialog/UploadDialogContainer.tsx

```diff
@@ -29,11 +29,17 @@ import CloseIconRounded from '@mui/icons-material/CloseRounded';
 import DialogBody from '../DialogBody/DialogBody';
 import UppyDashboard from '../UppyDashboard';
 import useSiteUIConfig from '../../hooks/useSiteUIConfig';
-import { XHRUploadOptions } from '@uppy/xhr-upload';
 import useUpdateRefs from '../../hooks/useUpdateRefs';
+import { Meta, Body } from '@uppy/utils/lib/UppyFile';
+import { XHRUploadOptions as UppyXHRUploadOptions } from '@uppy/xhr-upload';
```

Comment:
Remember to add `type` to type imports from code files
```suggestion
import type { Meta, Body } from '@uppy/utils/lib/UppyFile';
import type { XHRUploadOptions as UppyXHRUploadOptions } from '@uppy/xhr-upload';
```